nidm_results_display.m

```diff
@@ -14,7 +14,10 @@
 
     %Record users choice and filepath.
     jsondoc=spm_jsonread(jsonfilepath);
-    [pathstr, str] = fileparts(jsonfilepath); 
+    [pathstr, str] = fileparts(jsonfilepath);
+    
+    %We expect the files for the nidm pack to be stored in a folder, of the
+    %same name, located next to the jsons folder.
```

Comment:
We want the viewer to work with NIDM packs (i.e. `.nidm.zip`) rather than a folder. Is this possible with the current version?